### PR TITLE
[Trivial] Add a comment on the use of prevector in script.

### PR DESCRIPTION
**Pull request by gmaxwell**

### CODE REVIEW

File: src/script/script.h [hunk 1]
```diff
@@ -377,6 +377,12 @@ class CScriptNum
     int64_t m_value;
 };
 
+/**
+ * We use a prevector for the script to reduce the considerable memory overhead
+ *  of vectors in cases where they normally contain a small number of small elements.
+ * Tests in October 2015 showed use of this reduced dbcache memory usage by 23%
+ *  and and made an initial sync 13% faster.
```
Comment (paveljanik) [inline src/script/script.h]: and and

**Review by kallewoof — APPROVED** [review]

**Comment by instagibbs** [comment]
ACK(and and aside), very helpful!

**Comment by sdaftuar** [comment]
ACK (apart from @paveljanik's nit)

**Comment by fanquake** [comment]
utACK post nit

File: src/script/script.h [hunk 2]
```diff
@@ -377,6 +377,12 @@ class CScriptNum
     int64_t m_value;
 };
 
+/**
+ * We use a prevector for the script to reduce the considerable memory overhead
```
Comment (promag) [inline src/script/script.h]: nanonit: ` * Use a prevector ...`

**Pull request merged**